openrpc_validator/src/lib.rs

```diff
@@ -115,17 +117,6 @@ impl FireboltOpenRpc {
         None
     }
 
-    fn get_result_property(
```

Comment:
This is what was generating a property map out of the result's name field in the schema, we don't want to do that. If there are no result properties just return `None`.